goal_src/jak2/engine/gfx/ocean/ocean.gc

```diff
@@ -1160,10 +1160,11 @@
     (let ((f0-5 1.0)
           (f30-0 1.0)
           )
-      (when (= (-> *bigmap* bigmap-index) 13)
-        (set! f0-5 0.333)
-        (set! f30-0 0.75)
-        )
+      ;; TODO - need bigmap
+      ;; (when (= (-> *bigmap* bigmap-index) 13)
```

Comment:
can we leave this code the same, and just wrap it in `(when (nonzero? *bigmap*)`?
This way if we forget to restore it once we finish bigmap, it will work